src/library/Pagination/Pagination.js

```diff
@@ -0,0 +1,247 @@
+/* @flow */
+import React, { PureComponent } from 'react';
+import { createStyledComponent } from '../styles';
+import { withTheme } from '../themes';
+import Flex from '../Flex';
+import Pages from './Pages';
+import PageJumper from './PageJumper';
+import _PageSizer from './PageSizer';
+
+type Props = {
+  /**
+   * The category representing items to be paginated; utilized in
+   * [page sizer](#page-sizer) caption
+   */
+  category?: string,
+  /** The currently displayed page */
+  currentPage: number,
+  /** Hide the page number buttons */
+  hidePageNumbers?: boolean,
+  /**
+   * Various messages and labels used by Pagination
+   * ([see example for more details](#rtl))
+   */
+  messages: Messages,
+  /** Called when current page is changed */
+  onPageChange: (currentPage: number) => void,
+  /** Called when [page size](#page-sizer) is changed */
+  onPageSizeChange?: (pageSize: number) => void,
+  /**
+   * Render a [page jumper](#page-jumper)
+   * ([TextInput component](/components/text-input)); enables the user to change
+   * the current page on number entry
+   */
+  pageJumper?: boolean,
```

Comment:
`pageJumper` as a boolean prop feels strange.  I would expect it to take a component or a render function.  I would recommend to rename it to something like `showPageJumper`.  Of course that begs the question of whether `hidePageNumbers` should be renamed to `showPageNumbers` for consistency.

Reply:
Yes, agreed. The title "Hide Page Numbers" seems to contradict our example then; does "Previous & Next" sound ok?